src/GraphQL/Types/PercentageType.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Worksome\Number\GraphQL\Types;
+
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\Type;
+use Worksome\Number\Percentage;
+
+final class PercentageType extends ObjectType
+{
+    public function __construct()
+    {
+        parent::__construct([
+            'name' => 'Percentage',
+            'description' => 'A generic representation of a percentage.',
+            'fields' => [
+                'isZero' => [
+                    'type' => Type::nonNull(Type::boolean()),
+                    'description' => 'Whether the percentage is zero.',
+                ],
+                'isNegative' => [
+                    'type' => Type::nonNull(Type::boolean()),
+                    'description' => 'Whether the percentage is negative.',
+                ],
+                'isNegativeOrZero' => [
+                    'type' => Type::nonNull(Type::boolean()),
+                    'description' => 'Whether the percentage is negative or zero.',
+                ],
+                'isPositive' => [
+                    'type' => Type::nonNull(Type::boolean()),
+                    'description' => 'Whether the percentage is positive.',
+                ],
+                'isPositiveOrZero' => [
+                    'type' => Type::nonNull(Type::boolean()),
+                    'description' => 'Whether the percentage is positive or zero.',
+                ],
+                'toString' => [
+                    'type' => Type::nonNull(Type::string()),
+                    'description' => 'Retrieve the percentage as a string representation.',
```

Comment:
What does this specifically mean? Does it mean it will be 40% will be returned as `40` or `40%`

Reply:
also how is 40,30% represented, with comma or dot